api4/apitestlib.go

```diff
@@ -523,19 +523,19 @@ func (me *TestHelper) CreateChannelWithClientAndTeam(client *model.Client4, chan
 	return rchannel
 }
 
-func (me *TestHelper) CreatePost() *model.Post {
-	return me.CreatePostWithClient(me.Client, me.BasicChannel)
+func (th *TestHelper) CreatePost() *model.Post {
+	return th.CreatePostWithClient(th.Client, th.BasicChannel)
 }
 
-func (me *TestHelper) CreatePinnedPost() *model.Post {
-	return me.CreatePinnedPostWithClient(me.Client, me.BasicChannel)
+func (th *TestHelper) CreatePinnedPost() *model.Post {
+	return th.CreatePinnedPostWithClient(th.Client, th.BasicChannel)
 }
 
-func (me *TestHelper) CreateMessagePost(message string) *model.Post {
-	return me.CreateMessagePostWithClient(me.Client, me.BasicChannel, message)
+func (th *TestHelper) CreatethssagePost(message string) *model.Post {
```

Comment:
Typo here

Your sed replace had a bug ;) Instead of replace `me`, I would suggest replacing `me ` and `me.`. Much more robust.

Reply:
Oops, Thanks @agnivade! Sorry about that. I'll resolve it.